app/api/vector.py

```
@@ -0,0 +1,181 @@
+        
+    except HTTPException:
+        raise
+    except Exception as e:
+        logger.error(f"배치 임베딩 생성 실패: {e}")
+        raise HTTPException(status_code=500, detail=f"배치 임베딩 생성 실패: {str(e)}")
+
+
+async def _process_batch_embeddings(force_recreate: bool, batch_size: int):
+    """배치 임베딩 처리 - 간소화 버전"""
+    from app.core.database import AsyncSessionLocal
+    from app.core.dependencies import get_recommendation_service
```

Ruff (0.11.9)

Redefinition of unused `get_recommendation_service` from line 11

Remove definition: `get_recommendation_service`

(F811)